Makefile

```diff
@@ -59,10 +59,14 @@ endef
 
 TEST_TIMEOUT := 20m
 
+# TODO: INTEG_TEST should be functional tests
 INTEG_TEST_ROOT        := ./host
 INTEG_TEST_XDC_ROOT    := ./host/xdc
 INTEG_TEST_NDC_ROOT    := ./host/ndc
 
+INTEGRATION_TEST_ROOT := ./common/persistence/tests
```

Comment:
Nit: DB_INTEGRATION_TEST_ROOT

Reply:
It also include ES tests and any future dependency integration tests. So I want to just use integration tests.